flexget/plugins/sites/piratebay.py

```diff
@@ -51,6 +48,8 @@ class UrlRewritePirateBay(object):
             {
                 'type': 'object',
                 'properties': {
+                    'domain': {'type': 'string', 'default': DOMAIN},
```

Comment:
Couldn't it just be URI instead? I don't see the benefit in splitting it up.

Reply:
We could, but probably would have to drop the support for the `torrents.`-subdomain support in the `url_match` statement (or do some dirty hack there)